scripts/cli/environment.config.js

```diff
@@ -1,58 +1,66 @@
 // PM2 config file
-let chain = "ci-dev";
+let chain = "ci-local";
+let polymesh_bin = "../../target/release/polymesh";
+let common = "--wasm-execution compiled " +
+  //  "--db-cache=3000 " +
+  "--pruning=archive " +
+  "--rpc-methods=unsafe --rpc-external --ws-external " +
+  "--rpc-cors all --no-prometheus --no-telemetry --no-mdns " +
+  "--validator --chain " + chain + " ";
+// Use node-key parameter for the primary node.
+let primary = " --node-key 0000000000000000000000000000000000000000000000000000000000000001 " +
+  common;
+// The peer nodes need to connect to the primary.
+let peer = " --bootnodes /ip4/127.0.0.1/tcp/30334/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp " +
```

Comment:
For my own knowledge, how was the `12D3...` calculated?

Reply:
That specific value is a known peer ID for Alice derived from the specified  `--node-key`

These links might be of interest.
https://docs.libp2p.io/concepts/peer-id/

https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md

Reply:
I think I grabbed those two values from a blog/tutorial about running a multi-node setup.  Since this multi-node setup is just for developers doing testing, it doesn't need to be secure/secret.

Also nodes will print out their libp2p peer ids on startup, so that is one way to grab that value for the other peer nodes to connect too.

Reply:
Also FYI, the `run.sh` script will delete the previous DB files for those nodes.  So each stop/run will reset the multi-node network back to genesis block.

Reply:
Thanks, the links were informative and I didn't know this was printed on start up.

Reply:
Yes as Neopallium said you can get a peer ID by starting a node and checking the console log. As in this case a node key was specified for Alice it will have the above peer ID. First time a node is run, if it doesn't have a node key, it is generated and place in the keystore so it can be read and used for every subsequent start up. You can also use something like subkey to generate a node key and get the associated peer ID and place in a nodes keystore. https://docs.substrate.io/v3/tools/subkey/#generating-node-keys